src/index.css

```diff
@@ -1,70 +1,285 @@
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@300;500;600;700&display=swap');
```

Comment:
- download font and link using the below url
- https://stackoverflow.com/questions/41676054/how-to-add-fonts-to-create-react-app-based-projects